cordova-lib/src/cordova/metadata/ubuntu_parser.js

```diff
@@ -109,8 +110,8 @@ ubuntu_parser.prototype.update_manifest = function() {
         content += '\nIcon=qmlscene';
         console.warn('missing icon element in config.xml');
     }
-    fs.writeFileSync(path.join(this.path, 'manifest.json'), JSON.stringify(manifest));
-    fs.writeFileSync(path.join(this.path, 'cordova.desktop'), content);
```

Comment:
Would you replace this with packageManager.save or just in cases when JSON.stringify is used??